### PR TITLE
created api for llms, created README_ZH.md

### PR DESCRIPTION
**Pull request by z-zeechung**

### CODE REVIEW

**Comment by z-zeechung** [comment]
To add your llm api, view qianfan.py as an example

File: llm_api/llm_api.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    api = openai
+elif api == "claude":
+    from claude import claude
+    api = claude
+elif api == "qwen":
+    from qwen import qwen
+    api = qwen
+elif api == "qianfan":
+    from qianfan import qianfan
+    api = qianfan
```
Comment (neubig) [inline llm_api/llm_api.py:16]: For this, maybe we could use [litellm](https://github.com/BerriAI/litellm)? It gives an interface to many models so we don't need to implement it ourselves.

Comment (z-zeechung) [reply llm_api/llm_api.py:16]: that saves a deal of work

**Comment by AbhisekOmkar** [comment]
Yeah its good suggestion

File: llm_api/llm_api.py (hunk 1 above)
Comment (z-zeechung) [reply llm_api/llm_api.py]: i add some from litellm. the rest don't have embedding, which is important for knowledge base

**Comment by heum-ilgugim** [comment]
PR #17에 흡수됨. PR #17 (feat/ci-docker-patch-improvements)에서 lint/mypy/Docker/settings_router 전체 수정이 완료되었으며 2026-05-02 main에 merge됨.

**Pull request closed without merging**